modules/auth0/tenant/README.md

```diff
@@ -42,11 +42,11 @@ in Terraform. Follow the
 [Auth0 provider documentation](https://registry.terraform.io/providers/auth0/auth0/latest/docs/guides/quickstart) to
 create a Machine to Machine application.
 
-:::tip Machine to Machine App Name
-
-Use the Context Label format for the machine name for consistency. For example, `acme-plat-gbl-prod-auth0-provider`.
-
-:::
+> [!TIP]
+>
+> Machine to Machine App Name
```

Comment:
```suggestion
> #### Machine to Machine App Name
```

Reply:
Without converting anything docusaurus recognized the title with `####`!

This is what's in the mdx formatted for docusaurus:
```
:::tip

#### Machine to Machine App Name

Use the Context Label format for the machine name for consistency. For example, `acme-plat-gbl-prod-auth0-provider`.

:::
```

And this is the result:
![CleanShot 2024-08-07 at 12 42 24@2x](https://github.com/user-attachments/assets/11712957-b918-4c46-b35b-a4861d224a4e)

Reply:
and here's the GH style:
![CleanShot 2024-08-07 at 12 45 31@2x](https://github.com/user-attachments/assets/cdcd2878-94bd-4683-9f60-ec445b7745fe)